migrations/20240319001421_is_read.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE subscriber_notification ADD COLUMN is_read BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Probably `is_read` should be added to the index as well.

Reply:
Good catch, added in [fb7e55a](https://github.com/WalletConnect/notify-server/pull/423/commits/fb7e55ac15bde59da3833e788a0fe4a7143feff7)